src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -425,7 +425,10 @@ public function sendData($data)
         } catch (Exception $exception) {
             if ($eventDispatcher) {
                 // Log any errors with the request.
-                $eventDispatcher->dispatch(Constants::OMNIPAY_REQUEST_ERROR, new ErrorEvent($exception));
+                $eventDispatcher->dispatch(
+                    Constants::OMNIPAY_REQUEST_ERROR,
+                    new ErrorEvent($exception, $this)
```

Comment:
hmm might have made more sense for $this to come before $exception, so it lines up with the others. but oh well...

Reply:
To me it makes sense for it to come after because this is an `ErrorEvent` so the error should come first.